lib/rucio/core/scope.py

```diff
@@ -43,11 +43,13 @@ def add_scope(scope, account, session=None):
         new_scope.save(session=session)
     except IntegrityError as e:
         if match('.*IntegrityError.*ORA-00001: unique constraint.*SCOPES_PK.*violated.*', e.args[0]) \
-           or match('.*IntegrityError.*1062, "Duplicate entry.*for key.*', e.args[0]) \
+           or match('.*IntegrityError.*Duplicate entry.*for key.*', e.args[0]) \
```

Comment:
The error message from pymysql is slightly different and did not get catched

Reply:
this match happens in a lot of places, is it fixed everywhere?

Reply:
seems like in most places the correct regex is used. There is one wrong regex in core/account which I will change.